scripts/test-custom-ca.sh

```
@@ -0,0 +1,454 @@
+DEV_DIR="${ROOT_DIR}/docker/dev"
+CERT_DIR="${DEV_DIR}/certs"
+NGINX_DIR="${DEV_DIR}/nginx-test"
+COMPOSE_BASELINE="${DEV_DIR}/docker-compose.yaml"
+COMPOSE_CUSTOM_CA="${DEV_DIR}/docker-compose.custom-ca-example.yaml"
+COMPOSE_NGINX_TEST="${NGINX_DIR}/docker-compose.nginx-test.yaml"
+
+# Test configuration
+NGINX_PORT=8443
+CHECKMATE_PORT=52345
+HEALTH_ENDPOINT="http://localhost:$CHECKMATE_PORT/api/v1/health"
+TEST_URL="https://host.docker.internal:$NGINX_PORT"
```

Shellcheck (0.10.0)

[warning] TEST_URL appears unused. Verify use (or export if used externally).

(SC2034)